cmd/components.go

```diff
@@ -23,21 +24,32 @@ var ComponentsCmd = &cobra.Command{
 	Short: "List all Dapr components. Supported platforms: Kubernetes",
 	Run: func(cmd *cobra.Command, args []string) {
 		if kubernetesMode {
-			err := kubernetes.PrintComponents(componentsName, componentsOutputFormat)
+			if allNamespaces {
+				resourceNamespace = meta_v1.NamespaceAll
+			}
+			err := kubernetes.PrintComponents(componentsName, resourceNamespace, componentsOutputFormat)
```

Comment:
This must be reported as a breaking change since the default behavior is now to list the default namespace or not all.

Reply:
Following k8s conventions only the default namespace is default :) otherwise a specific all flag needs to be specified. I am ok with this change as long as we clearly mention it as a breaking change and if we are ok with having a breaking change for this.